gspread/worksheet.py

```diff
@@ -1681,9 +1695,11 @@ def append_rows(
         params = {
             "valueInputOption": value_input_option,
             "insertDataOption": insert_data_option,
-            "includeValuesInResponse": include_values_in_response,
         }
 
+        # Not an Enum, does not pass `extract_enum_values`
+        params["includeValuesInResponse"] = include_values_in_response
```

Comment:
can this go back in the `params` object?

Reply:
Yes or can. Good catch.

Reply:
done